README.md

```diff
@@ -245,6 +245,7 @@ To be added
 ## Projects using `eta`
 
 - [Docusaurus v2](https://v2.docusaurus.io): open-source documentation framework that uses Eta to generate a SSR build
+- [swagger-typescript-api](https://github.com/acacode/swagger-typescript-api): open-source typescript api codegenerator from Swagger. Use `ETA` as codegenerator by templates  
```

Comment:
Can you capitalize the sentence? Also the name of this project is Eta, not `ETA`.

Reply:
Fixed.

Reply:
Can you capitalize Open source? also remove the extra hyphen please.

Reply:
@shadowtime2000 fixed!